client/src/main/java/com/streamr/client/rest/Json.java

```diff
@@ -16,4 +20,12 @@ private Json() {}
         .add(Address.class, new AddressJsonAdapter().nullSafe())
         .add(new InstantJsonAdapter());
   }
+
+  public static String withValue(String json, String key, String value) throws IOException {
+    JsonAdapter<Object> adapter = Json.newMoshiBuilder().build().adapter(Object.class);
+    @SuppressWarnings("unchecked")
+    Map<String, Object> jsonObject = (Map<String,Object>) adapter.fromJson(json);
```

Comment:
```suggestion
    ParameterizedType type = Types.newParameterizedType(Map.class, String.class, Object.class);
    JsonAdapter<Map<String, Object>> adapter = Json.newMoshiBuilder().build().adapter(type);
    Map<String, Object> jsonObject = adapter.fromJson(json);
```

I'm not sure if it compiles, but it would be better without suppress warnings unless absolutely necessary.

Reply:
Added error handling and unit test